src/features/content-answers/ContentAnswers.tsx

```diff
@@ -10,12 +11,17 @@ const ContentAnswers = () => {
     const [isModalOpen, setIsModalOpen] = useState(false);
 
     const handleClick = () => {
-        setIsModalOpen(!isModalOpen);
+        setIsModalOpen(true);
+    };
+
+    const handleClose = () => {
+        setIsModalOpen(false);
     };
 
     return (
-        <div>
+        <div className="ContentAnswers">
             <ContentAnswersOpenButton onClick={handleClick} />
+            <ContentAnswersModal isModalOpen={isModalOpen} onRequestClose={handleClose} />
```

Comment:
nit: the "Modal" part is implied because the component name already says modal so it's not needed in the prop name
```suggestion
            <ContentAnswersModal isOpen={isModalOpen} onRequestClose={handleClose} />
```